apps/yarn.lock

```diff
@@ -3859,10 +3859,15 @@ async@0.9.0:
   version "0.9.0"
   resolved "https://registry.yarnpkg.com/async/-/async-0.9.0.tgz#ac3613b1da9bed1b47510bb4651b8931e47146c7"
 
+async@^0.9.0:
+  version "0.9.2"
+  resolved "https://registry.yarnpkg.com/async/-/async-0.9.2.tgz#aea74d5e61c1f899613bf64bda66d4c78f2fd17d"
+  integrity sha512-l6ToIJIotphWahxxHyzK9bnLR6kM4jJIIgLShZeqLY7iboHoGkdgFl7W2/Ivi4SkMJYGKqW8vSuk0uKUj6qsSw==
+
 async@^1.5.2:
   version "1.5.2"
   resolved "https://registry.yarnpkg.com/async/-/async-1.5.2.tgz#ec6a61ae56480c0c3cb241c95618e20892f9672a"
-  integrity sha1-7GphrlZIDAw8skHJVhjiCJL5Zyo=
+  integrity sha512-nSVgobk4rv61R9PUSDtYt7mPVB2olxNR5RWJcAsH676/ef11bUZwvu7+RGYrYauVdDPcO519v68wRhXQtxsV9w==
```

Comment:
For my learning –– do you know how it's determined which sha function is used for these? No worries if not

Reply:
this is kind of a guess based on some light reading, but i think it depends on two things:
1. the version of yarn. with yarn 2.0, sha512 became the default, but was available in earlier versions with the [unsafe-disable-integrity-migration](https://classic.yarnpkg.com/en/docs/yarnrc#toc-unsafe-disable-integrity-migration) option
2. the sha that the package has available. i don't feel like i really understand this piece 😄 but someone mentions it in this [stack overflow](https://stackoverflow.com/questions/71377672/use-of-sha1-and-md5-in-integrity-of-package-in-yarn-lock). maybe this piece depends on the version of yarn/NPM that the package publisher is using?